[DOCS] Update Add Validation Data or Expectation Suites to a Checkpoint (#8931)
great-expectations/great_expectations: docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md

</Prerequisites>

## Add an Expectation Suite to the Checkpoint

> To add a second Expectation Suite (in this example we add ``users.error``) to your Checkpoint configuration, update the validations in your Checkpoint.  The resulting configuration will look like this:

Author: ```suggestion
To add a second Expectation Suite (in the following example, `users.error`) to your Checkpoint configuration, you update the Validations in your Checkpoint.  The configuration appears similar to this example:
```

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.py add_expectation_suite"
```

> ## Add validation data to the Checkpoint

Author: ```suggestion
## Add Validation data to the Checkpoint
```

> In the previous example, the Validation you added was paired with the same Batch as the original Expectation Suite.  However, you can also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> when you add an Expectation Suite.  Adding multiple Validations with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> is demonstrated in the following example:

Author: ```suggestion
In the previous example, you added a Validation and it was paired with the same Batch as the original Expectation Suite.  However, you can also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> when you add an Expectation Suite.  Adding multiple Validations with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> is shown in the following example:
```

> In the previous example, the validation you added with your Expectation Suite was paired with the same Batch of data as the original Expectation Suite.  However, you may also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> (and thus different Batches of data) when you add an Expectation Suite.  The flexibility of easily adding multiple Validations of Batches of data with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> can be demonstrated using the following example:

Author: ```suggestion
In the previous example, the Validation you added was paired with the same Batch as the original Expectation Suite.  However, you can also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> when you add an Expectation Suite.  Adding multiple Validations with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> is demonstrated in the following example:
```

```python name="tests/integration/docusaurus/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.py add_validation"
```

> In this Checkpoint configuration, the Expectation Suite `users.warning` runs against the `batch_request` with the results processed by the Actions specified in the `action_list`. Similarly, the Expectation Suite `users.error` runs against the `batch_request` with the results processed by the actions specified in the `action_list`. In addition, the Expectation Suite `users.delivery` runs against the `batch_request` with the results processed by actions in the Validation `action_list` and the `action_list`.

Author: ```suggestion
In this Checkpoint configuration, the Expectation Suite `users.warning` runs against the `batch_request` and the results are processed by the Actions specified in the `action_list`. Similarly, the Expectation Suite `users.error` runs against the `batch_request` and the results processed by the actions specified in the `action_list`. In addition, the Expectation Suite `users.delivery` runs against the `batch_request` and the results are processed by the actions in the Validation `action_list` and its `action_list`.
```

> According to this Checkpoint configuration, the Expectation Suite ``users.warning`` is run against the ``batch_request`` with the results processed by the Actions specified in the top-level ``action_list``. Similarly, the Expectation Suite ``users.error`` is run against the ``batch_request`` with the results also processed by the actions specified in the top-level ``action_list``. In addition, the top-level Expectation Suite ``users.delivery`` is run against the ``batch_request`` with the results processed by the union of actions in the validations ``action_list`` and in the top-level ``action_list``.

Author: ```suggestion
In this Checkpoint configuration, the Expectation Suite `users.warning` runs against the `batch_request` with the results processed by the Actions specified in the `action_list`. Similarly, the Expectation Suite `users.error` runs against the `batch_request` with the results processed by the actions specified in the `action_list`. In addition, the Expectation Suite `users.delivery` runs against the `batch_request` with the results processed by actions in the Validation `action_list` and the `action_list`.
```
